src/Libraries/CoreNodeModels/Input/ColorPalette.cs

```diff
@@ -99,32 +143,15 @@ protected override void DeserializeCore(XmlElement element, SaveContext context)
         [IsVisibleInDynamoLibrary(false)]
         public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode> inputAstNodes)
         {
-            var av = AstFactory.BuildIntNode(Convert.ToInt32(dsColor.Alpha));
-            var ar = AstFactory.BuildIntNode(Convert.ToInt32(dsColor.Red));
-            var ag = AstFactory.BuildIntNode(Convert.ToInt32(dsColor.Green));
-            var ab = AstFactory.BuildIntNode(Convert.ToInt32(dsColor.Blue));
+            var av = AstFactory.BuildIntNode(Convert.ToInt32(DsColor.Alpha));
+            var ar = AstFactory.BuildIntNode(Convert.ToInt32(DsColor.Red));
+            var ag = AstFactory.BuildIntNode(Convert.ToInt32(DsColor.Green));
+            var ab = AstFactory.BuildIntNode(Convert.ToInt32(DsColor.Blue));
 
             var colorNode = AstFactory.BuildFunctionCall(
                     new Func<int, int, int, int, DSColor>(DSColor.ByARGB), new List<AssociativeNode> { av, ar, ag, ab });
 
             return new[] { AstFactory.BuildAssignment(GetAstIdentifierForOutputIndex(0), colorNode) };
         }
-
-        /// <summary>
-        ///     Override string representation of color in watch node.
-        /// </summary>
-        /// <returns></returns>
-        public override string ToString()
-        {
-            return string.Format("Color(Alpha = {3}, Red = {0}, Green = {1}, Blue = {2})", dsColor.Alpha, dsColor.Red, dsColor.Green, dsColor.Blue);
-        }
-
-        /// <summary>
-        ///     Indicates whether node is input node
-        /// </summary>
-        public override bool IsInputNode
-        {
-            get { return false; }
```

Comment:
good point, not sure why this was removed, I'll check add it back - THOUGH, now that we have UI nodes 🃏